src/modules/customer/containers/HappyUserPage/components/formFields/CheckboxGroup.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../WizardForm/messages';
+
+import '../../components/WizardForm/WizardForm.css';
+
+class CheckboxGroup extends React.PureComponent {
+  checkboxGroup() {
+    let { options, input } = this.props;
+
+    return options
+      .map((option, index) => {
+        return (
+          <div className="wizard-form__checkbox" key={index}>
+            <label>
+              <input
+                type="checkbox"
+                name={`${input.name}[${index}]`}
+                value={option.name}
+                checked={input.value.indexOf(option.name) !== -1}
+                onChange={event => {
```

Comment:
You define a function inside render. This means the component will rerender every time. Such functions should be moved outside render to `this.onChange = event => {}`

onChange={this.onChange}